grails-app/services/org/pih/warehouse/data/DataService.groovy

```diff
@@ -684,7 +685,7 @@ class DataService {
                     datePicked       : requisition?.picklist?.datePicked ? "${formatDate.format(requisition?.picklist?.datePicked)}" : "",
 
                     issuedBy         : requisition.issuedBy,
-                    dateIssued       : requisition.dateIssued ? "${formatDate.format(requisition.dateIssued)}" : "",
+                    dateIssued       : requisition.status == RequisitionStatus.ISSUED ? "${formatDate.format(requisition.shipment?.expectedShippingDate)}" : "",
```

Comment:
The `requisition.dateIssued` field is definitely wrong since we never populate it, so I'm ok with removing that. But how did you determine you should use `requisition.shipment.expectedShippingDate`? That might not be precise enough. Instead we should try to use the transaction.transactionDate. If we don't have a way of accessing the `transactionDate` through the requisition or shipment then we might need to figure that out. For example, I know both have a hasMany to a list of transactions but it might be complicated and expensive (joins) to get the transactionDate from that list. So let me know if you think there's a better way of doing it (maybe populating the requisition.dateIssued after the stock movement has been issued) or writing custom SQL to pull all of the data we need in a single query.

Reply:
@jmiranda It is possible to access transactionDate from requisition but what is more problematic is the fact that there is only transaction created on sending shipment if origin is a depot (for inbound it is created during receiving process) and I assume that we should also display date issued of requisitions that have origin other than depot. Assigning dateIssued on sending stockMovement seems to be the best approach but what with those requisitions that has already been send? Maybe for empty requisition.dateIssued we could display requisition.shipment.expectedShippingDate then?